gpu-operator/release-notes.rst

```diff
@@ -31,8 +31,96 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
 
    GPU Operator beta releases are documented on `GitHub <https://github.com/NVIDIA/gpu-operator/releases>`_. NVIDIA AI Enterprise builds are not posted on GitHub.
 
+
 ----
 
+.. _v24.3.0:
+
+24.3.0
+======
+
+.. _v24.3.0-new-features:
+
+New Features
+------------
+
+
```

Comment:
We are missing one bullet here -- added support for Kubernetes version 1.30 (since our QA tested 1.30 in their final round of testing). We will also need to update the platform support page in various tables.

Reply:
I updated to 1.30 is all places except the [KubeVirt table](https://nvidia.github.io/cloud-native-docs/review/pr-44/gpu-operator/latest/platform-support.html#support-for-kubevirt-and-openshift-virtualization).  PLMK if I should update there too.